mdm/hibernate-core/src/main/java/com/gentics/mesh/database/HibernateTxImpl.java

```diff
@@ -409,7 +409,11 @@ public <T extends HibElement> T persist(T element) {
 
 	@Override
 	public <T extends HibElement> void delete(T element) {
-		em.remove(element);
+		if (em.contains(element)) {
+			em.remove(element);
+		} else {
+			forceDelete(element, "dbUuid", e -> e.getId(), false);
```

Comment:
Looks like they have messed up the entity manager again, so it may appear that the entity is detached/deleted. I'll look further into the reasons for that. As of now the fixes work, the consistency checks show no errors.